app/services/create_user.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: false
+
+# Creates user
+class CreateUser < ApplicationService
+  def initialize(user, cart)
+    @user = user
```

Comment:
Acho que deveriamos chamar dessa forma se são apenas parâmetros. Pois `@user` pode indicar que é uma instância de `User`.
```suggestion
    @user_params = user_params
```